src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+title: Add a web app manifest
+authors:
+  - petelepage
+date: 2018-11-05
+updated: 2020-02-14
+description: |
+  The web app manifest is a simple JSON file that tells the browser about your
+  web application and how it should behave when 'installed' on the user's mobile
```

Comment:
Does 'installed' need to be quoted. This is the first time I've seen it that way.

Reply:
Good point - fixed.